serverlessworkflow/modules/ROOT/pages/cloud/operator/build-workflow-images-with-tekton.adoc

```diff
@@ -0,0 +1,143 @@
+= Building Workflows Images with Tekton
+:compat-mode!:
+// Metadata:
+:description: Building Workflows Images with Tekton
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, openshift, containers, tekton, pipelines
+// links
+:kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/
```

Comment:
don't we have this env already set on another place?

Reply:
Remove, was copied from another page but are useless here